packages/blade/src/components/Alert/_decisions/decisions.md

```diff
@@ -0,0 +1,162 @@
+# Alert
+
+Alerts are messages that communicate information to users about any significant changes or explanations inside the system in a **prominent way**.
+
+This document outlines the API of `Alert` component.
+
+<img src="./alerts-thumbnail.png" width="380" alt="" />
+
+## Design
+
+- [Figma - Alerts](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=6824%3A0)
+
+> **Note**
+>
+> The following needs work:
+>
+> 1. color on banking theme in high contrast
+> 2. a11y contrast ratio for notice alerts in high contrast
+> 3. the borderless variant mentioned in full bleed layout shouldn't have any borders and border radii
+
+## API
+
+Annotated component with props:
+
+![Alert breakdown](./alert-breakdown.png)
+
+Sample usage:
+
+```jsx
+import { Alert, Link } from '@razorpay/blade';
+
+<Alert title="International Payments Only">
+  <Alert.Description>
+    Currently you can only accept payments in international currencies using PayPal. You cannot
+    accept payments in INR (₹) using PayPal. <Link href="https://razorpay.com">Know More</Link>
+  </Alert.Description>
+  <Alert.PrimaryAction
+    onClick={() => {
+      // do something
+    }}
+  >
+    Primary Action
+  </Alert.PrimaryAction>
+  <Alert.SecondaryAction
+    onClick={() => {
+      // do something
+    }}
+    href="https://razorpay.com"
+  >
+    Link
+  </Alert.SecondaryAction>
+</Alert>;
+```
+
+### Alert
+
+We'll expose an `Alert` component with the following API:
+
+| Prop          | Type                                     | Default     | Description                                                                                | Required |
+| ------------- | ---------------------------------------- | ----------- | ------------------------------------------------------------------------------------------ | -------- |
+| `children`    | `string`, Alert sub components           | `undefined` | Body content                                                                               | ✅       |
+| title         | `string`                                 | `undefined` | A brief heading                                                                            |          |
+| isDismissable | `boolean`                                | `true`      | Shows a dismiss button                                                                     |          |
```

Comment:
we probably might need an onClick handler for this to maybe capture some analytics?